pkgs/development/libraries/libiff/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "libiff";
+  version = "0.1";
```

Comment:
Since there are no tags, the version should be named `0-unstable-yyyy-mm-dd` (https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)

Reply:
Addressed in https://github.com/NixOS/nixpkgs/pull/308273